### ISSUE TITLE
Tray icon menu appears way above tray icon

### ISSUE THREAD

**Issue by wailmerpailmer** [mumble-voip/mumble#3999]
**Describe the bug**
The tray icon menu appears way above the tray icon when right clicked.

**Steps to Reproduce**
1. Use KDE
2. Right click the tray icon

**Expected behavior**
The menu should appear directly above the tray icon.

**Screenshots**
![image](https://user-images.githubusercontent.com/46793924/77004555-26aa7d00-6957-11ea-9e08-545f9ec21c17.png)

**Desktop (please complete the following information):**
 - OS: KDE Plasma 5.18
 - Version: Mumble 1.3

**Comment by Krzmbrzl** [comment]
Yeah I also noticed that one before. Also on KDE. Just didn't come around creating a ticket before, so thanks for taking action :)

Do you happen to know of anyone not using KDE having the same problem?

**Comment by wailmerpailmer** [comment]
Nobody that I know who uses it on Windows has this problem.

**Comment by Krzmbrzl** [comment]
Okay that means it is probably a Linux issue (and probably KDE specific). Will have to set up a vm to test with a different desktop environment...
